### DISCUSSION TITLE
Need OpenBangla Keyboard for Windows

### DISCUSSION THREAD

**Discussion by sakibstime** [OpenBangla/OpenBangla-Keyboard#312] [Ideas]
Hello,
I'm a regular user of OpenBangla Keyboard. I'm using it on my Linux distro PC. But, I need to work on Windows also where I'm missing this software.

I don't find any installation file to install it on Windows PC. You may get more audience who will love to use this software on their Windows PC. 

Please consider releasing the Windows version.

Thank you.

**Comment by abuhuraira0099** [comment]
Windows version is still under development. You can use Omicronlab's Avro software or Codepotro's Borno Keyboard to get similar keyboard layouts and features on Windows.  [Edit : English]

**Reply by sakibstime** [reply]
Thanks for your suggestion. I'm using Avro right now. I wait for the OpenBangla keyboard,

**Reply by yeasin2002** [reply]
> Windows version is still under development. You can use Omicronlab's Avro software or Codepotro's Borno Keyboard to get similar keyboard layouts and features on Windows. [Edit : English]

ডেভেলপমেন্ট প্রসেস কি এখনও আছে? আমি এখনও windows এর জন্য কোণ  installation গাইড দেখছি না

**Reply by mominul** [reply]
@yeasin2002 উইন্ডোজ পোর্টের ডেভেলপমেন্ট প্রায় শেষের পথে 🎉


https://github.com/user-attachments/assets/343a758c-8362-4a7a-961c-fc7efe72dfa4

**Reply by sakibstime** [reply]
Waiting for this. 😃

**Reply by yeasin2002** [reply]
Waiting for this 2
